sourcegraph/about-replatform
# Add pricing page and components

Adds pricing page and related components and closes #62. Does **not** bring over `PricingFreeTierPopover` and `HoverablePopover` as they were not need/used (updated in #48)


## To Test
- Check out the branch and update local dependencies with `npm ci`
- Head to the `/pricing` page and make sure everything looks as expected
- Test on static builds
- Ensure no console warnings from anything to do with this PR
